source/Orchestrator/test/test_exec_ssm_doc.py

```diff
@@ -60,95 +57,79 @@ def test_exec_runbook(mocker):
                 "aws/securityhub/ProductName": "Security Hub",
                 "aws/securityhub/CompanyName": "AWS",
                 "aws/securityhub/annotation": "AWS Config evaluated your resources against the rule. The rule did not apply to the AWS resources in its scope, the specified resources were deleted, or the evaluation results were deleted.",
-                "aws/securityhub/FindingId": "arn:aws:securityhub:us-east-1::product/aws/securityhub/arn:aws:securityhub:us-east-1:111111111111:subscription/aws-foundational-security-best-practices/v/1.0.0/AutoScaling.1/finding/635ceb5d-3dfd-4458-804e-48a42cd723e4"
+                "aws/securityhub/FindingId": "arn:aws:securityhub:us-east-1::product/aws/securityhub/arn:aws:securityhub:us-east-1:111111111111:subscription/aws-foundational-security-best-practices/v/1.0.0/AutoScaling.1/finding/635ceb5d-3dfd-4458-804e-48a42cd723e4",
```

Comment:
Was this a linting thing? I've often had problems with having a comma on the last element of a collection (json doesn't allow it)

Reply:
This isn't json, it's Python. [Here](https://github.com/asottile/add-trailing-comma#multi-line-method-invocation-style----why) are some reasons why. I defer to my formatter, which in this case is black. Haven't set up linting for python in this project yet, that will be a massive diff

Reply:
fair enough, just wanted to check